ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -75,6 +77,8 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
                 params.append('label', label);
             });
         }
+        params.append('minStartedAt', this.state.minStartedAt.toISOString());
```

Comment:
I don't see how this can work. This is not supported my `minStartedAt`.

Reply:
I forgot to remove them.